core/chains/evm/logpoller/orm.go

```diff
@@ -335,93 +312,74 @@ func (o *DbORM) SelectLogsByBlockRange(start, end int64) ([]Log, error) {
 	return logs, nil
 }
 
-// SelectLogsByBlockRangeFilter finds the logs in a given block range.
+// SelectLogs finds the logs in a given block range.
 func (o *DbORM) SelectLogs(start, end int64, address common.Address, eventSig common.Hash, qopts ...pg.QOpt) ([]Log, error) {
-	args, err := newQueryArgsForEvent(o.chainID, address, eventSig).
-		withStartBlock(start).
-		withEndBlock(end).
-		toArgs()
-	if err != nil {
-		return nil, err
-	}
-	var logs []Log
-	err = o.q.WithOpts(qopts...).SelectNamed(&logs, `
-		SELECT * FROM evm.logs 
-			WHERE evm_chain_id = :evm_chain_id 
-			AND address = :address
-			AND event_sig = :event_sig  
-			AND block_number >= :start_block 
-			AND block_number <= :end_block
-			ORDER BY (block_number, log_index)`, args)
-	if err != nil {
-		return nil, err
-	}
-	return logs, nil
+	//SELECT * FROM evm.logs
+	//			WHERE evm_chain_id = :evm_chain_id
+	//			AND address = :address
+	//			AND event_sig = :event_sig
+	//			AND block_number >= :start_block
+	//			AND block_number <= :end_block
+	//			ORDER BY (block_number, log_index)
+	filter := NewBasicAndFilter(
```

Comment:
Can't we use `NewBlockRangeFilter()` here?

Reply:
Fixed ;) Keep in mind it's poc, so some parts of the code might be hacky a bit and not prod-ready ;)